deploy/operator/hypershift/deploy_hypershift_cluster.sh

```diff
@@ -0,0 +1,123 @@
+export ASSISTED_PULLSECRET_JSON="${ASSISTED_PULLSECRET_JSON:-${PULL_SECRET_FILE}}"
+export ASSISTED_PRIVATEKEY_NAME="${ASSISTED_PRIVATEKEY_NAME:-assisted-ssh-private-key}"
+export EXTRA_BAREMETALHOSTS_FILE="${EXTRA_BAREMETALHOSTS_FILE:-/home/test/dev-scripts/ocp/ostest/extra_baremetalhosts.json}"
+export SPOKE_KUBECONFIG_SECRET="${SPOKE_KUBECONFIG_SECRET:-${ASSISTED_CLUSTER_NAME}-kubeconfig}"
+export SPOKE_CONTROLPLANE_AGENTS="${SPOKE_CONTROLPLANE_AGENTS:-1}"
+export CLUSTER_SUBNET="${CLUSTER_SUBNET_V4}"
+export CLUSTER_HOST_PREFIX="${CLUSTER_HOST_PREFIX_V4}"
+export EXTERNAL_SUBNET="${EXTERNAL_SUBNET_V4}"
+export SERVICE_SUBNET="${SERVICE_SUBNET_V4}"
+export PUBLIC_CONTAINER_REGISTRIES="${PUBLIC_CONTAINER_REGISTRIES:-quay.io}"
+
+set -o nounset
+set -o pipefail
+set -o errexit
+set -o xtrace
+
+playbooks_dir=${__dir}/playbooks
+
+# Hypershift CLI needs access to the kubeconfig, pull-secret and public SSH key
+function hypershift() {
+  podman run -it --net host --rm --entrypoint /usr/bin/hypershift -v $KUBECONFIG:/root/.kube/config -v $ASSISTED_PULLSECRET_JSON:/root/pull-secret.json -v /root/.ssh/id_rsa.pub:/root/.ssh/id_rsa.pub $HYPERSHIFT_IMAGE "$@"
+}
+
+# Get agents on spoke cluster
+function get_agents() {
+  oc --kubeconfig $SPOKE_KUBECONFIG get agent -n $SPOKE_NAMESPACE --no-headers
+}
+
+# Ensure namespaces exist
+oc get namespace "${SPOKE_NAMESPACE}" || oc create namespace "${SPOKE_NAMESPACE}"
+oc get namespace "${HYPERSHIFT_AGENT_NS}" || oc create namespace "${HYPERSHIFT_AGENT_NS}"
+
+# We need a storage for etcd of the hosted cluster
+oc patch storageclass assisted-service -p '{"metadata": {"annotations":{"storageclass.kubernetes.io/is-default-class":"true"}}}'
+
+# Update assisted-service-operator Subscription
+oc -n $ASSISTED_NAMESPACE patch subscription assisted-service-operator --type merge \
+    -p '{"spec":{"config":{"env":[{"name":"PUBLIC_CONTAINER_REGISTRIES","value":"'${PUBLIC_CONTAINER_REGISTRIES}'"}]}}}'
+
+echo "Installing HyperShift using upstream image"
+hypershift install --hypershift-image $HYPERSHIFT_IMAGE --namespace hypershift
+wait_for_pods "hypershift"
+
+echo "Creating HostedCluster"
+oc get hostedcluster ${ASSISTED_CLUSTER_NAME} -n ${HYPERSHIFT_AGENT_NS} || \
+    hypershift create cluster agent --name $ASSISTED_CLUSTER_NAME --base-domain redhat.example --pull-secret /root/pull-secret.json \
+        --ssh-key /root/.ssh/id_rsa.pub --agent-namespace $HYPERSHIFT_AGENT_NS --namespace $HYPERSHIFT_AGENT_NS
+
+echo "Wait for a running hypershift cluster with no worker nodes"
+#wait_for_pods "$SPOKE_NAMESPACE"
+wait_for_condition "nodepool/$ASSISTED_CLUSTER_NAME" "Ready" "10m" "$HYPERSHIFT_AGENT_NS"
+wait_for_condition "hostedcluster/$ASSISTED_CLUSTER_NAME" "Available" "10m" "$HYPERSHIFT_AGENT_NS"
+
+echo "Extract spoke kubeconfig"
+oc extract -n $HYPERSHIFT_AGENT_NS secret/$ASSISTED_CLUSTER_NAME-admin-kubeconfig --to=- > /tmp/$ASSISTED_CLUSTER_NAME-kubeconfig
+export SPOKE_KUBECONFIG=/tmp/$SPOKE_KUBECONFIG_SECRET
```

Comment:
shouldn't this work as well?
```suggestion
export SPOKE_KUBECONFIG=/tmp/$SPOKE_KUBECONFIG_SECRET
oc extract -n $HYPERSHIFT_AGENT_NS secret/$ASSISTED_CLUSTER_NAME-admin-kubeconfig --to=$SPOKE_KUBECONFIG
```

Reply:
ok
and what about changing ``--to=- > /tmp/$ASSISTED_CLUSTER_NAME-kubeconfig`` to ``--to=-/tmp/$ASSISTED_CLUSTER_NAME-kubeconfig``?

Reply:
doesn't work:/ it's just a copy-paste from: https://github.com/openshift/assisted-service/blob/master/deploy/operator/capi/deploy_capi_cluster.sh#L148